blockly-core/core/ui/css.js

```diff
@@ -132,7 +95,10 @@ Blockly.Css.setCursor = function(cursor, opt_svg) {
  */
 Blockly.Css.CONTENT = [
   '.blocklyDraggable {',
-    // Placeholder for cursor rule. Must be first rule (index 0).
+  '  cursor: url(/blockly/media/handopen.cur) 8 5, auto;',
+  '}',
+  '.dragging, .dragging .blocklyDraggable {',
```

Comment:
Your change depends on the `.blocklyDraggable` being a child of the root SVG which you've added the `.dragging` class to, but there are a few cases where this function can be called without `opt_svg` and you won't have set that class.  Have you tracked down those edge cases?